benchmark/misc/v8-bench.js

```diff
@@ -12,7 +12,11 @@ global.print = function(s) {
 
 global.load = function (x) {
   var source = fs.readFileSync(path.join(dir, x), 'utf8');
+  // deps/v8/benchmarks/regexp.js breaks console.log() because it clobbers
+  // the RegExp global,  Restore the original when the script is done.
+  var $RegExp = global.RegExp;
   vm.runInThisContext(source, x);
```

Comment:
the 2nd argument `x` is filename. 
According to api docs, 2nd argument is option.

https://iojs.org/api/vm.html#vm_vm_runinthiscontext_code_options

```javascript
vm.runInThisContext(source, {filename: x});
```

or remove 2nd arg.

```javascript
vm.runInThisContext(source);
```